src/test/gas-consumption/gas.spec.ts

```diff
@@ -294,4 +295,26 @@ describe("benchmarks", () => {
         const gasUsedRaw = measureGas(deployRawResult.transactions);
         expect(gasUsedRaw).toMatchSnapshot("gas used raw deploy");
     });
+
+    const variants: [string, "GetSqrt" | "GetSqrtViaGeomMean"][] = [
+        ["sqrt via geom mean", "GetSqrtViaGeomMean"],
+        ["sqrt", "GetSqrt"],
+    ];
+
+    it.each(variants)("benchmark %s", async (name, message) => {
+        const instance = blockchain.openContract(await Sqrt.fromInit());
+
+        const sendResult = await step(name, () =>
+            instance.send(
+                treasure.getSender(),
+                { value: toNano(1) },
+                { $$type: message, value: 100n },
```

Comment:
Let's also try a few different values: `0`, `MAX_INT`, typical Toncoin values, for instance